includes/wc-core-functions.php

```diff
@@ -631,8 +631,7 @@ function get_woocommerce_currencies() {
 					'USD' => __( 'United States (US) dollar', 'woocommerce' ),
 					'UYU' => __( 'Uruguayan peso', 'woocommerce' ),
 					'UZS' => __( 'Uzbekistani som', 'woocommerce' ),
-					'VEF' => __( 'Venezuelan bol&iacute;var', 'woocommerce' ),
-					'VES' => __( 'Bol&iacute;var soberano', 'woocommerce' ),
+					'VES' => __( 'Venezuelan bol&iacute;var', 'woocommerce' ),
```

Comment:
Hi, I don't think we can just remove the currency, as that might have strange effects on stores that have it set as their currency. What I'd recommend is that we rename the VEF to something like Venezuelan Bolivar (pre-2019) or some other accurate name and VES to the Venezuelan bolivar. What do you think?

Reply:
Sorry for the very late reply, @peterfabian! Yes, I agree with this approach :)